homeassistant/components/media_player/mpd.py

```diff
@@ -225,11 +248,15 @@ def media_previous_track(self):
 
     def play_media(self, media_type, media_id, **kwargs):
         """Send the media player the command for playing a playlist."""
-        _LOGGER.info(str.format("Playing playlist: {0}", media_id))
-        if media_type == MEDIA_TYPE_PLAYLIST:
```

Comment:
Use the original if/else structure here. The goal is to allow a playlist request to go through even if it isn't in our cached list of playlists. If `media_id` is not in `self.playlists`, set `self.currentplaylist = None`.